hack/aad.sh

```diff
@@ -55,8 +56,19 @@ EOF
     fi
 
     cat <<EOF
-AZURE_CLIENT_ID=$AZURE_CLIENT_ID
-AZURE_CLIENT_SECRET=$AZURE_CLIENT_SECRET
+AZURE_AAD_CLIENT_ID=$AZURE_AAD_CLIENT_ID
+AZURE_AAD_CLIENT_SECRET=$AZURE_AAD_CLIENT_SECRET
+EOF
+    ;;
+
+app-update)
+    if [[ "$#" -ne 2 ]]; then usage; fi
+    AZURE_AAD_CLIENT_SECRET=$(uuidgen)
+    az ad app update --id $AZURE_AAD_CLIENT_ID --reply-urls "https://$2/oauth2callback/Azure%20AD" --key-type password --password $AZURE_AAD_CLIENT_SECRET
+
```

Comment:
Change of URL is not allowed with an old secret. You need to refresh it too. Nice security feature I would say.